p2p/auth.py

```diff
@@ -48,7 +49,7 @@ def authenticate(username=None, password=None, token=None, auth_url=None):
 try:
     from django.contrib.auth.models import User
 
-    class P2PBackend:
+    class P2PBackend(object):
```

Comment:
FWIW, this doesn't do anything in Python 3, only in Python 2.

Reply:
Removed